client/src/components/Contexts/WalletStatusContext.tsx

```diff
@@ -52,7 +52,14 @@ function WalletContextProvider(props: { children: any }) {
         if (web3 && chainId && account) {
             const c = await API.initialize(chainId, account)
             setContracts(c)
-            const owner = await c.behodler.Behodler.primary().call({ from: account })
+            const owner = (async () => {
+                try {
+                    return await c.behodler.Behodler.primary().call({ from: account })
+                } catch (e) {
+                    console.error('Error calling Behodler "primary" contract', e)
+                    return '';
+                }
+            })();
```

Comment:
`c.behodler.Behodler.primary().call({ from: account })` is another case of contract call failing on Ropsten with `MetaMask - RPC Error: execution reverted` error. This makes me believe that I either use an incorrect contract address for Behodler or that some of the required contracts are not deployed.